docs/ldml/tr35-general.html

```diff
@@ -2491,6 +2514,9 @@ <h3>6.<span class='changed'>5</span> <a name="Unit_Sequences" href="#Unit_Sequen
 &lt;listPattern type="unit-narrow"&gt;
 &lt;listPattern type="unit-short"&gt;
 </pre>
+    <p class="changed">In such sequences, decimal values are typically only used with
```

Comment:
Sorry, I was on my phone, and it somehow didn't record an additional comment.

Please change to:

In such as sequence, decimal fractions are typically only displayed for the last element of the sequence, if at all.

Reply:
Yes, that is better, thanks, made that change too